### PR TITLE
[IMP] sales: menuselection fix

### PR DESCRIPTION
**Pull request by meng-odoo**
Fixed a menuselection error and deleted instances of second-person pov
Closes task [3116083](https://www.odoo.com/web#id=3116083&cids=3&model=project.task&view_type=form)

### CODE REVIEW

**Comment by meng-odoo** [comment]
@StraubCreative this is ready for your review.

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 1]
```diff
@@ -2,19 +2,19 @@
 Multiple sales teams
 ====================
 
-Odoo lets you manage several sales teams, departments, or channels, each with their own unique
-sales processes, using *Sales Teams*.
+Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
+their own unique sales processes.
 
 
 Create a new Sales Team
```
Comment (StraubCreative) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:9]: Would use [sentence case](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings) here, just like the H1.
```suggestion
Create a new sales team
```

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
 =======================
 
-To create a new Sales Team, go to :menuselection:`CRM --> Sales --> Teams` then click
+To create a new Sales Team, go to :menuselection:`CRM --> Configuration --> Sales Teams` then click
 :guilabel:`Create`.
```
Comment (StraubCreative) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:13]: `Sales Team` doesn't need to be capitalized here, as it was already introduced as a term in the overview and it's not a proper noun.
```suggestion
To create a new sales team in Odoo, go to :menuselection:`CRM --> Configuration --> Sales Teams`
then click :guilabel:`Create`.
```

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
 
 On the creation page, set an :guilabel:`Email Alias` to automatically generate a lead/opportunity
-for this Sales Team every time a message is sent to that unique email address. You can also choose
-whether to accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or
+for this Sales Team every time a message is sent to that unique email address. Choose whether to
```
Comment (StraubCreative) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:16]: Lowercase
```suggestion
for this sales team every time a message is sent to that unique email address. Choose whether to
```

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 4]
```diff
@@ -2,19 +2,19 @@
...
+accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or
 :guilabel:`Followers Only`.
 
 Set a :guilabel:`Domain` to assign leads/opportunities to this Sales Team based on specific
```
Comment (StraubCreative) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:20]:
```suggestion
Set a :guilabel:`Domain` to assign leads/opportunities to this sales team based on specific
```

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 5]
```diff
@@ -43,8 +43,8 @@ access all of the pipelines that they need to.
 Sales Team dashboard
```
Comment (StraubCreative) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:43]:
```suggestion
Sales team dashboard
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @meng-odoo 

Good PR. Short and sweet and the RST's clean 👍 

I had only two comments:
- `sales-team-creation.png` might be just a touch too big (at ~1300px)
- `Sales Teams` is not a proper noun so I wouldn't capitalize it except when: 
    - first introducing it as a term/feature in the overview, and 
    - whenever we're calling attention to a UI element that's called `Sales Teams`

I highlighted a couple instances below, however there are much more throughout the doc. 
Can you take a look?

**Comment by meng-odoo** [comment]
Hi @StraubCreative, I fixed the capitalization of "sales team" throughout the doc as requested. I also realized that all of the screenshots were of v14, so I updated all of them to v15. This is ready for your final review.

This PR should be forward-ported to v16 documentation, please. (We can leave v14 alone, there is no menuselection error in that doc.)

Thanks

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 6]
```diff
@@ -2,61 +2,61 @@
...
-Add members to a Sales Team
+Add members to a sales team
 ---------------------------
 
-To add team members, click :guilabel:`Add` under the Assignment tab when editing the Sales Team's
-configuration page. Select a salesperson from the dropdown menu or create new salesperson. Set a
-maximum number of leads that can be assigned to this salesperson in a 30-day period to ensure that
-they do not overwork.
+To add team members, click :guilabel:`Add` under the :guilabel:`Members` tab when editing the sales
+team's configuration page. Select a salesperson from the dropdown menu or create new salesperson.
```
Comment (StraubCreative) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:32]: Compound modifier gets a hyphen.
```suggestion
team's configuration page. Select a salesperson from the drop-down menu or create new salesperson.
```

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 7]
```diff
@@ -2,61 +2,61 @@
...
-a part of will appear as dashboard tiles.
+To view the sales team dashboard, go to :menuselection:`CRM --> Sales --> Teams`. Odoo users will
+see any teams that they are a part of as dashboard tiles.
 
-Each tile gives an overview of the Sales Team's open opportunities, quotations, sales orders, and
+Each tile gives an overview of the sales team's open opportunities, quotations, sales orders, and
 expected revenue, as well as a bar graph of new opportunities per week and an invoicing progress
 bar.
 
 .. image:: multi_sales_team/sales-team-overview.png
```
Comment (StraubCreative) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:53]: Resize + compress

**Review by StraubCreative — APPROVED** [review]
Hi @meng-odoo 👋 
Approving with a couple quick changes I'll push up momentarily.
Thank you!

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**